models/intermediate/int_ga4_export__sessions_report.sql

```diff
@@ -1,14 +1,10 @@
-{{
+{{ 
```

Comment:
Is this model used at all? It looks like it's not referenced by any other downstream models?

Reply:
That sounds like a plan to me!

Reply:
done